--- contrib/halog/halog.c.orig	2014-07-25 01:56:07.000000000 -0500
+++ contrib/halog/halog.c	2014-07-25 15:24:56.996876765 -0500
@@ -462,7 +462,7 @@ int convert_date(const char *field)
 {
 	unsigned int h, m, s, ms;
 	unsigned char c;
-	const char *b, *e;
+	const char *e;
 
 	h = m = s = ms = 0;
 	e = field;
@@ -477,7 +477,6 @@ int convert_date(const char *field)
 	}
 
 	/* hour + ':' */
-	b = e;
 	while (1) {
 		c = *(e++) - '0';
 		if (c > 9)
@@ -488,7 +487,6 @@ int convert_date(const char *field)
 		goto out_err;
 
 	/* minute + ':' */
-	b = e;
 	while (1) {
 		c = *(e++) - '0';
 		if (c > 9)
@@ -499,7 +497,6 @@ int convert_date(const char *field)
 		goto out_err;
 
 	/* second + '.' or ']' */
-	b = e;
 	while (1) {
 		c = *(e++) - '0';
 		if (c > 9)
@@ -512,7 +509,6 @@ int convert_date(const char *field)
 	/* if there's a '.', we have milliseconds */
 	if (c == (unsigned char)('.' - '0')) {
 		/* millisecond second + ']' */
-		b = e;
 		while (1) {
 			c = *(e++) - '0';
 			if (c > 9)
@@ -535,7 +531,7 @@ int convert_date_to_timestamp(const char
 {
 	unsigned int d, mo, y, h, m, s;
 	unsigned char c;
-	const char *b, *e;
+	const char *e;
 	time_t rawtime;
 	static struct tm * timeinfo;
 	static int last_res;
@@ -622,7 +618,6 @@ int convert_date_to_timestamp(const char
 	}
 
 	/* hour + ':' */
-	b = e;
 	while (1) {
 		c = *(e++) - '0';
 		if (c > 9)
@@ -633,7 +628,6 @@ int convert_date_to_timestamp(const char
 		goto out_err;
 
 	/* minute + ':' */
-	b = e;
 	while (1) {
 		c = *(e++) - '0';
 		if (c > 9)
@@ -644,7 +638,6 @@ int convert_date_to_timestamp(const char
 		goto out_err;
 
 	/* second + '.' or ']' */
-	b = e;
 	while (1) {
 		c = *(e++) - '0';
 		if (c > 9)
@@ -686,10 +679,10 @@ void truncated_line(int linenum, const c
 
 int main(int argc, char **argv)
 {
-	const char *b, *e, *p, *time_field, *accept_field, *source_field;
+	const char *b, *p, *time_field, *accept_field, *source_field;
 	const char *filter_term_code_name = NULL;
 	const char *output_file = NULL;
-	int f, last, err;
+	int f, last;
 	struct timer *t = NULL;
 	struct eb32_node *n;
 	struct url_stat *ustat = NULL;
@@ -941,7 +934,7 @@ int main(int argc, char **argv)
 				}
 			}
 
-			e = field_stop(time_field + 1);
+			field_stop(time_field + 1);
 			/* we have field TIME_FIELD in [time_field]..[e-1] */
 			p = time_field;
 			f = 0;
@@ -965,17 +958,15 @@ int main(int argc, char **argv)
 				}
 			}
 
-			e = field_stop(time_field + 1);
+			field_stop(time_field + 1);
 			/* we have field TIME_FIELD in [time_field]..[e-1], let's check only the response time */
 
 			p = time_field;
-			err = 0;
 			f = 0;
 			while (!SEP(*p)) {
 				tps = str2ic(p);
 				if (tps < 0) {
 					tps = -1;
-					err = 1;
 				}
 				if (++f == 4)
 					break;
@@ -1701,7 +1692,7 @@ void filter_count_ip(const char *source_
 void filter_graphs(const char *accept_field, const char *time_field, struct timer **tptr)
 {
 	struct timer *t2;
-	const char *e, *p;
+	const char *p;
 	int f, err, array[5];
 
 	if (!time_field) {
@@ -1712,7 +1703,7 @@ void filter_graphs(const char *accept_fi
 		}
 	}
 
-	e = field_stop(time_field + 1);
+	field_stop(time_field + 1);
 	/* we have field TIME_FIELD in [time_field]..[e-1] */
 
 	p = time_field;
