--- contrib/iprange/iprange.c.orig	2014-06-27 14:01:26.020159479 -0500
+++ contrib/iprange/iprange.c	2014-06-27 14:01:42.149275554 -0500
@@ -111,7 +111,7 @@ static void usage(const char *argv0)
 		"\n", argv0);
 }
 
-main(int argc, char **argv)
+int main(int argc, char **argv)
 {
 	char line[MAXLINE];
 	int l, lnum;
@@ -198,4 +198,5 @@ main(int argc, char **argv)
 			convert_range(sa, da, he, NULL);
 		}
 	}
+	exit(0);
 }
